ws/v1.md

```diff
@@ -0,0 +1,123 @@
+
+Minimum request:
+```
+{
+    "type": "subscribe",
+    "channel": "orders",
+    "requestId": "123e4567-e89b-12d3-a456-426655440000"
+}
+```
+This will subscribe to all new orders and order state changes in the orderbook.
+
+Filtered request:
+```
+{
+    "type": "subscribe",
+    "channel": "orders",
+    "requestId": "123e4567-e89b-12d3-a456-426655440000",
+    "payload": {
+        "makerAssetData": "0xf47261b04c32345ced77393b3530b1eed0f346429d",
+        "takerAssetData": "0x0257179264389b814a946f3e92105513705ca6b990",
+        "networkId": 42,
+    }
+}
+```
+This will subscribe to all new Kovan orders and Kovan order state changes in the orderbook with `makerAssetData` and `takerAssetData` equal to the values specified in the subscribe request.
+
+**Parameters**
+
+General:
+*   `requestId`: a string uuid that will be sent back by the server in response messages so the client can appropriately respond when multiple subscriptions are made
+*   `networkId`: the Ethereum network id to which you'd like to subscribe. Default is 1 (mainnet). (optional)
+   
+Networks and their Ids:
+
+| Network Id| Network Name |
+| ----------| ------------ |
+| 1         | Mainnet      |
+| 42        | Kovan        |
+| 3         | Ropsten      |
+| 4         | Rinkeby      |
+
+Filter parameters (optional): 
+*   `makerAssetProxyId` : returns orders where the maker asset is of certain [asset proxy id](https://0xproject.com/docs/0x.js#types-AssetProxyId) (example: `0xf47261b0` for `ERC20`, `0x02571792` for `ERC721`)
+*   `takerAssetProxyId`:  returns orders where the taker asset is of certain [asset proxy id](https://0xproject.com/docs/0x.js#types-AssetProxyId)(example: `0xf47261b0` for `ERC20`, `0x02571792` for `ERC721`)
+*   `makerAssetAddress`: subscribes to new orders where the contract address for the maker asset matches the value specified
+*   `takerAssetAddress`: subscribes to new orders where the contract address for the taker asset matches the value specified
+
+Order specific parameters (optional):
+*   `makerAssetData`: subscribes to new orders with the specified makerAssetData
+*   `takerAssetData`: subscribes to new orders with the specified takerAssetData
+*   `traderAssetData`: subscribes to new orders where either makerAssetData or takerAssetData has the value specified
+
+While there is some redundancy in supporting maker/takerAssetType, maker/takerAssetAddress, and maker/takerAssetData, they are actually all needed. For example, you cannot query "all ERC712 orders", or "all Cryptokitties orders", or "all ERC20 orders" with just maker/takerAssetData and need the other query params to do so.
+
+#### Response messages
+
+##### Update
+
+This message is sent whenever the relayer receives a new order, or when the `remainingFillableAmount` of an order changes. The scenarios where an update is necessary include:
```

Comment:
Can we clarify that this is with respect to the takerAssetAmount? I feel that it could be interpreted takerAssetAmount or makerAssetAmount, and I'm just assuming it's takerAssetAmount because of how the contract internals are implemented.